build/ci/community/wait_demo_pr_checks.sh

```diff
@@ -1,33 +1,32 @@
 #!/bin/bash -xe
 set +o pipefail
 
+gh_pr_checks_command (){
```

Comment:
if we run the command, I suggest naming the function accordingly:
```suggestion
run_gh_pr_checks_command (){
```

Reply:
isn't `gh_pr_checks` enough?

Reply:
I agree with the article, though I think in this case, the function is being used more like a command alias than a function.
I think this distinction is important, otherwise we might wrap all the commands we use with a long, low-level-detail named functions of the form `run_X_command`.
if you still disagree, at least consider the shorter `get_pr_checks`

Reply:
`get_pr_checks` sounds great.
If you want to continue with the discussion about naming, I suggest having a session with Arbel.

Reply:
@matancarmeli7 let's use `get_pr_checks` here